crates/modelardb_server/src/main.rs

```diff
@@ -15,7 +15,7 @@
 
 //! Implementation of ModelarDB's main function.
 
-#![allow(clippy::too_many_arguments)]
+#![allow(clippy::too_many_arguments, clippy::unconditional_recursion)]
```

Comment:
Clippy warnings should be fixed if at all possible. If it is not possible at all, the place in the code that emit that warning should be annotated with `clippy::unconditional_recursion` so it is clear what part of the code the warning is suppressed from. By placing `clippy::unconditional_recursion` here, all `clippy::unconditional_recursion` warnings are suppressed.

Reply:
As I mentioned fixing it required considerable changes, so I placed `clippy::unconditional_recursion` in the respective file.

Reply:
Ok, I have added a task to fix it to the backlog.